app/src/components/Table/Table.scss

```diff
@@ -83,6 +83,10 @@
       color: white;
       font-weight: 700;
     }
+
+    &.-error {
+        color: red;
```

Comment:
Please use $negative-red for color scheme consistency.

Reply:
Will remove when I can.